src/article_metrics/api_v2_urls.py

```diff
@@ -6,6 +6,8 @@
     # article-level metrics
     re_path(r'^ping$', views.ping, name='ping'),
     re_path(r'^article/(?P<msid>\d+)/(?P<metric>(citations|downloads|page-views))$', views.article_metrics, name='alm'),
+    re_path(r'^article/(?P<msid>\d+)/(?P<metric>(citations|downloads|page-views))/version/(?P<version>\d+)$',
```

Comment:
minor: if we only support `citations`, might it be better to only match on that?